contrib/src/sceneeditor/seSession.py

```diff
@@ -208,36 +211,36 @@ def oobe(self):
             # Remove any transformation on the models arc
             self.oobeVis.clearMat()
             # Make oobeCamera be a sibling of wherever camera is now.
-            cameraParent = self.camera.getParent()
+            cameraParent = base.direct.camera.getParent()
             # Prepare oobe camera
             self.oobeCamera.reparentTo(cameraParent)
-            self.oobeCamera.iPosHpr(self.trueCamera)
+            self.oobeCamera.setPosHpr(self.trueCamera.getPos(), self.trueCamera.getHpr())
             # Put camera under new oobe camera
             base.cam.reparentTo(self.oobeCamera)
             # Position a target point to lerp the oobe camera to
             self.cameraControl.camManipRef.setPos(
                 self.trueCamera, Vec3(-2,-20, 5))
             self.cameraControl.camManipRef.lookAt(self.trueCamera)
-            t = self.oobeCamera.lerpPosHpr(
-                Point3(0), Vec3(0), 2.0,
+            t = self.oobeCamera.posHprInterval(
+                2.0, Point3(0), Vec3(0),
                 other = self.cameraControl.camManipRef,
-                task = 'manipulateCamera',
+                name = 'manipulateCamera',
                 blendType = 'easeInOut')
             # When move is done, switch to oobe mode
             t.uponDeath = self.beginOOBE
 
     def beginOOBE(self, state):
         # Make sure we've reached our final destination
-        self.oobeCamera.iPosHpr(self.cameraControl.camManipRef)
-        self.camera = self.oobeCamera
+        self.oobeCamera.setPosHpr(self.cameraControl.camManipRef.getPos(), self.cameraControl.camManipRef.getHpr())
```

Comment:
The correct way to replace iPosHpr in this instance is:
`self.oobeCamera.setPosHpr(self.cameraControl.camManipRef, 0, 0, 0, 0, 0, 0)`

This handles the case correctly where the two nodes in question have different parents.